frontend/src/GameLogic/FlipCard.tsx

```diff
@@ -0,0 +1,123 @@
+import {useEffect, useState} from "react";
+import {GameCard} from "./GameCard.ts";
+import axios from "axios";
+import './FlipCard.css';
+import {IconButton, Stack} from "@mui/material";
+import {Link} from "react-router-dom";
+import HomeRoundedIcon from "@mui/icons-material/HomeRounded";
+import ReplayRoundedIcon from "@mui/icons-material/ReplayRounded";
+
+type Props = {
+    gameSize: "small" | "medium" | "large"
+    gameName: string
+    colorStyle: string
+    colorStyle2: string
+    colorStyle3: string
+}
+export default function FlipCard(props: Props) {
+
+    const [, setBoardId] = useState<string>("")
```

Comment:
Why is there no BoardId?

Reply:
SonarLint gave me a warning there, because I don't use it. I had to delete it 🙂